services/app/src/static/style/header.css

```diff
@@ -26,3 +26,46 @@
   font-size: 16px;
   color: #212121;
 }
+
+/*-----CopyIcon.jsx-----*/
```

Comment:
Move the styles that are CopyIcon specific into a separate file that would be stored next to CopyIcons.jsx